tests/architecture/test_components.py

```
@@ -4,34 +4,33 @@
+from pathlib import Path
+
+# Architecture refactor tests - now enabled since components are implemented!
+
+from promptix.core.components import (
+    PromptLoader,
+    VariableValidator,
+    TemplateRenderer,
+    VersionManager,
+    ModelConfigBuilder
+)
+from promptix.core.exceptions import (
```

Flake8 (7.2.0)

[error] 'promptix.core.exceptions.UnsupportedClientError' imported but unused

(F401)